src/editors/multiselect.js

```diff
@@ -8,12 +8,17 @@ JSONEditor.defaults.editors.multiselect = JSONEditor.AbstractEditor.extend({
     var items_schema = this.jsoneditor.expandRefs(this.schema.items || {});
 
     var e = items_schema["enum"] || [];
+	var eTitles = items_schema.options.enum_titles || items_schema["enum"] || [];
+	
+	
     this.option_keys = [];
+	this.option_titles = [];
     for(i=0; i<e.length; i++) {
       // If the sanitized value is different from the enum value, don't include it
       if(this.sanitize(e[i]) !== e[i]) continue;
 
       this.option_keys.push(e[i]+"");
+	  this.option_titles.push(eTitles[i]+"");
```

Comment:
Update: I changed the tab/space indentation with the github site manual edit. Tell me If I can do anything else to advance the merge.